### PR TITLE
Added MDN Link for Aarray.Prototype.sort() for further reference

### LINKED ISSUE #9404: Description missing Algorithm/Details.
Challenge [Sort Arrays with sort](https://www.freecodecamp.com/challenges/sort-arrays-with-sort) has an issue.

The description uses

```
var array = [1, 12, 21, 2];
array.sort(function(a, b) {
  return a - b;
});
```

to "sort the elements from smallest to largest number".

But nowhere is this algorithmic concept described.

### PR DESCRIPTION
**Pull request by rajkthakur**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) them into one commit).
- [x] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [ ] Small bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace 9404 with an issue no): Closes #9404 
#### Description

I have added a MDN link which describes the array sort method in javascript

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -136,6 +136,7 @@
   },
   "devDependencies": {
     "browser-sync": "^2.9.12",
+    "gulp": "^3.9.1",
```
Comment (atjonathan) [inline package.json]: This shouldn't be added.

**Comment by rajkthakur** [comment]
@atjonathan do I need to remove line 139??

**Comment by atjonathan** [comment]
@rajkthakur, correct :+1:

**Comment by pawatzaza** [comment]
sw

**Comment by pawatzaza** [comment]
ok

File: seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json [hunk 2]
```diff
@@ -444,7 +444,8 @@
         "If no compare (callback) function is passed in, it will convert the values to strings and sort alphabetically.",
         "Here is an example of using sort with a compare function that will sort the elements from smallest to largest number:",
         "<blockquote>var array = [1, 12, 21, 2];<br>array.sort(function(a, b) {<br>&nbsp;&nbsp;return a - b;<br>});</blockquote>",
-        "Use <code>sort</code> to sort <code>array</code> from largest to smallest."
+        "Use <code>sort</code> to sort <code>array</code> from largest to smallest.",
+        "<a href='https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort' target='_blank'>Further explanation on <code>sort</code> method</a>"
```
Comment (atjonathan) [inline seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json]: ~~`"<a href='https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort' target='_blank'>Further explanation on <code>sort</code> method</a>"`~~ `"**Further explanation on the <code>.sort()</code> method can be found** <a href='https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort' target='_blank'>here</a>`

Comment (raisedadead) [inline seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json]: @atjonathan You might want to check the line `found**` ??

Comment (atjonathan) [inline seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json]: @raisedadead, sorry just a markdown glitch :wink:, I fixed it now. Thanks for catching that :+1:

**Comment by raisedadead** [comment]
#### :sparkles: :sparkles: **LGTM.** :sparkles: :sparkles: 

@rajkthakur Thanks a lot for the fix. And congratulations on your first contribution to Free Code Camp. Hope to see more in future!

**Comment by rajkthakur** [comment]
@atjonathan @raisedadead @QuincyLarson @camperbot this was my first ever contribution to any open source and to start with Freecodecamp is always special as I have been its part for some months now, although It was small but It was like breaking the jinx, a lot of thanks to all of you :)

**Comment by atjonathan** [comment]
@rajkthakur, No problem at all. Good luck :smile:

**Pull request merged**